resources/views/Customer/Components/Registration.twig

```diff
@@ -18,13 +11,24 @@
                 </div>
 
                 <div class="col-sm-6" v-if="!guestMode">
-                    <div class="input-unit" :class="{'no-bottom media-xs-d': modalElement}" id="passwordBox" data-validate="password">
-                        <input type="password" name="password" id="registration_password_input" v-model="password">
+                    <div class="input-unit" :class="{'no-bottom media-xs-d': modalElement}" id="passwordBox" data-validate="password" ref="passwordInput">
+                        <popper v-cloak :trigger="'focus'" :placement="'bottom'" ref="passwordHint">
```

Comment:
@felixdausch please change :...="'....'" to ...=""

https://vuejs.org/v2/guide/components.html#Passing-Data-with-Props